recipes/widgetsnbextension/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.0" %}
+
+package:
+    name: widgetsnbextension
+    version: {{ version }}
+
+source:
+    fn: widgetsnbextension-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/w/widgetsnbextension/widgetsnbextension-{{ version }}.tar.gz
```

Comment:
This URL works locally but not in the CIs. I don't know the first thing about re-directing URLs, but the new PyPI ones are terrible.